mmdeploy/codebase/mmseg/deploy/segmentation_model.py

```diff
@@ -106,7 +106,7 @@ def forward_test(self, imgs: torch.Tensor, *args, **kwargs) -> \
         """
         outputs = self.wrapper({self.input_name: imgs})
         outputs = self.wrapper.output_to_list(outputs)
-        outputs = [out.detach().cpu().numpy() for out in outputs]
+        outputs = [out.detach().cpu().numpy() for _, out in outputs]
```

Comment:
visualize  failed when exporting the model.

Reply:
fixed.